Rakefile

```diff
@@ -1,394 +1,12 @@
-  package_found = packager.update_from_s3
-  raise "No valid package found" unless package_found
-end
-
-def ensure_apps_package
-  # never download if we build our own
-  return if CDO.use_my_apps
-
-  packager = S3Packaging.new('apps', apps_dir, dashboard_dir('public/apps-package'))
-  package_found = packager.update_from_s3
-  raise "No valid package found" unless package_found
-end
-
-namespace :install do
-
-  # Create a symlink in the public directory that points at the appropriate blockly
-  # code (either the static blockly or the built version, depending on CDO.use_my_apps).
-  task :blockly_symlink do
-    make_blockly_symlink
-  end
-
-  task :hooks do
-    files = [
-        'pre-commit',
-        'post-checkout',
-        'post-merge',
-    ]
-    git_path = ".git/hooks"
-
-    files.each do |f|
-      path = File.expand_path("../tools/hooks/#{f}", __FILE__)
-      RakeUtils.ln_s path, "#{git_path}/#{f}"
-    end
-  end
-
-  task :apps do
-    if local_environment?
-      install_npm
-    end
-  end
-
-  task :code_studio do
-    if local_environment?
-      make_code_studio_symlink
-      ensure_code_studio_package
-      install_npm
-    end
-  end
-
-  task :dashboard do
-    if local_environment?
-      Dir.chdir(dashboard_dir) do
-        RakeUtils.bundle_install
-        puts CDO.dashboard_db_writer
-        RakeUtils.rake 'dashboard:setup_db'
-      end
-    end
-  end
-
-  task :pegasus do
-    if local_environment?
-      Dir.chdir(pegasus_dir) do
-        RakeUtils.bundle_install
-        RakeUtils.rake 'pegasus:setup_db'
-      end
-    end
-  end
-
-  tasks = []
-  #tasks << :blockly_core if CDO.build_blockly_core
-  tasks << :hooks if rack_env?(:development)
-  tasks << :blockly_symlink
-  tasks << :apps if CDO.build_apps
-  tasks << :code_studio if CDO.build_code_studio
-  tasks << :dashboard if CDO.build_dashboard
-  tasks << :pegasus if CDO.build_pegasus
-  task :all => tasks
-
-end
-task :install => ['install:all']
-
-# Commands to update built static asset packages
-namespace :update_package do
-
-  task :code_studio do
-    ensure_code_studio_package
-  end
-
-  task :apps do
-    ensure_apps_package
-  end
-
-end
-
-namespace :adhoc do
-  task :environment do
-    CDO.chef_local_mode = true
-    ENV['RAILS_ENV'] = ENV['RACK_ENV'] = CDO.rack_env = 'adhoc'
-    require 'cdo/aws/cloud_formation'
-  end
-  namespace :start do
-    task default: :environment do
-      AWS::CloudFormation.create_or_update
-    end
-    task cdn: :environment do
-      AWS::CloudFormation.create_or_update(true)
-    end
-  end
-  task start: ['start:default']
-  task stop: :environment do
-    AWS::CloudFormation.delete
-  end
-  task validate: :environment do
-    AWS::CloudFormation.validate
-  end
-end
+Rake::TaskManager.record_task_metadata = true
+Dir.glob('lib/rake/*.rake').delete_if{|rake| rake.include? 'generate_pdfs'}.each{|rake| import rake}
```

Comment:
yeah, I didn't like this either, so I moved `generate_pdfs` to pegasus/rake along with the other pegasus tasks (and refactored the code to be contained within a non-default rake task, so it would play nice with the others)